config/jobs/kubernetes/sig-scalability/sig-scalability-periodic-jobs.yaml

```diff
@@ -117,6 +117,7 @@ periodics:
       - --kubemark
       - --kubemark-nodes=100
       - --provider=gce
+      - --env=CL2_ENABLE_CLUSTER_OOMS_TRACKER=true
```

Comment:
What does OOM tracker do in kubemark? Does it really do something useful?

IIUC, this is watching events from kubemark cluster, where NPD is faked, so it's baiscally no-op.

Reply:
It still tracks components on the master.

Reply:
ok - good point.
That said, we should probably make that explicit somewhere (measurement doc?) [that it only works for master components and we're fine with it.]